go.mod

```diff
@@ -6,24 +6,29 @@ require github.com/spf13/pflag v1.0.5
 
 require (
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.19.0
-	k8s.io/apimachinery v0.25.0
-	sigs.k8s.io/kubebuilder/v3 v3.7.1-0.20220923133227-91a1afa8fe9e
+	github.com/onsi/gomega v1.24.2
+	k8s.io/apimachinery v0.26.1
```

Comment:
Why not 0.26.2?

Reply:
I updated it to 0.26.1 as we are doing this across all operator-framework repos.

Reply:
To keep it consistent across all of OF repos, I think it makes sense to bump it to `0.26.1`. `0.26.2` was released just 5 days ago and I've started the k8s bumps a little over a week ago. :)

Reply:
But, I updated it now.

Reply:
Actually, I think it would be ok to bump to latest which is `0.26.2`. I don't think it effects any other dependencies that way, so fine to use latest. I will switch to latest patch release as well for the upcoming PRs on k8s bump to 1.26.

Reply:
Yes, already updated.